### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by Mih-Mir**

### CODE REVIEW

File: Singleton/Program.cs [hunk 1]
```diff
@@ -1,12 +1,74 @@
...
+            amount = dbActions.CreateUpdateDelete(updateExpression);
+
+            var deleteExpression = "DELETE FROM Profiles WHERE Id=4";
+            amount = dbActions.CreateUpdateDelete(deleteExpression);
+
+            dbActions.CloseConnection();
+        }
+    }
+
+    public sealed class DbActions
```
Comment (AndreyYurashevich) [inline Singleton/Program.cs:30]: There is no actual singleton class in the solution. Try making DbActions class itself a singleton. Note that if DbActions becomes a singleton then there is no need for special logic for initialization of the SqlConnection

**Review by AndreyYurashevich — CHANGES_REQUESTED** [review]

**Review by sachenko-igor — CHANGES_REQUESTED** [review]
Follow Andrey's requirements

File: Singleton/Program.cs (hunk 1 above)
Comment (Mih-Mir) [reply Singleton/Program.cs:28]: Rewrite.

**Review by AndreyYurashevich — APPROVED** [review]
`SqlConnection _sqlConnection;` does not need to be static, but otherwise looks good to me.

**Pull request merged**